astro/src/content/docs/extend/code/password-hashes/custom-password-hashing.mdx

```diff
@@ -102,3 +103,25 @@ That may also be useful to you, as someone may already have written the hasher y
 ## Rehashing User Passwords
 
 <RehashingUserPasswords />
+
+## Deleting Plugins
+
+You should never delete an installed password hashing plugin unless you are certain that no users are using the hash provided by that plugin. If any users still have that `encryptionScheme` associated with their account, they will see undefined behavior when they try to login or change their password. This may include runtime exceptions.
+
+The only way to completely and safely delete a custom password hashing plugin is to ensure all user accounts have migrated off of the algorithm defined by the plugin.
+
+However, the hashing scheme is associated with each account and cannot be queried by the [User Search API](/docs/apis/users#search-for-users). You can view it for an individual user by navigating to <Breadcrumb>Users -> Your User ->Manage</Breadcrumb> and choosing the <Breadcrumb>Source</Breadcrumb> tab. But you can't query this value all users.
+
+If you have a plan with support and need to know which users are using a particular password hashing scheme, [open a support ticket](https://account.fusionath.io/account/support) so that the team can provide you with the query (if you are self-hosted) or run the query for you (if you are in FusionAuth Cloud).
```

Comment:
Should we just say contact support for further assistance and not mention the db query at all?

Reply:
I originally had that:

```
 so that the team can help you.
```

but wanted to provide a clue for all our community users. But I think you're right, it'll be simpler if we just ask them to file a support ticket. And if they don't have one, they might be able to read between the lines (or read the public schema).